src/pages/projectCreate/ProjectCreationWalletConnectionForm.tsx

```diff
@@ -126,7 +126,7 @@ export const ProjectCreationWalletConnectionForm = ({
                     w="full"
                     name="lightning-address"
                     type={'email'}
-                    placeholder={'satoshi@getalby.com'}
+                    placeholder={'runwithbitcoin@getalby.com'}
```

Comment:
Was this intentional to change the placeholder?

Reply:
Yes, this was very intentional. :D